.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,18 +4,17 @@ about: Create a report to help us improve
 title: ''
 labels: ''
 assignees: ''
-
 ---
-
  - [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
  - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).
 
 **Environment**
- - OS: [e.g. Debian, Windows, macOS...]
- - scrcpy version: [e.g. 1.12.1]
- - installation method: [e.g. manual build, apt, snap, brew, Windows release...]
- - device model:
- - Android version: [e.g. 10]
+ - **OS**: [e.g. Debian, Windows, macOS...]
+ - **Kernel** (if linux) `neofetch`
```

Comment:
We don't care about the client kernel version (and `neofetch` is not really a standard tool to get it).